### ISSUE TITLE
Sprite is broken due to DOCTYPE in source SVG files

### ISSUE THREAD

**Issue by Eireen** [kisenka/svg-sprite-loader#135]
Any occurrence of DOCTYPE in result sprite breaks its loading in browsers (FF 52, Chrome 48.0.2564.116).

The source files look like:

```xml
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 17.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
    width="9.414px" height="16.414px" viewBox="0 0 9.414 16.414" style="enable-background:new 0 0 9.414 16.414;"
    xml:space="preserve" fill="#748592">
<path class="st0" d="M9.414,15L8,16.414l-8-8l0.207-0.207L0,8l8-8l1.414,1.414L2.621,8.207L9.414,15z"/>
</svg>
```

My [webpack.config.js](https://gist.github.com/Eireen/b235013758311ceb79e7d8016a231bd3)

- Node.js version: v7.10.0
- webpack version: 2.6.1
- svg-sprite-loader version: 3.0.4
- OS type & version: Xubuntu 12.04

**Comment by kisenka** [comment]
Thanks for reporting, its known issue. As quick workaround you can use svgo-loader before svg-sprite-loader.

**Comment by kisenka** [comment]
@Eireen please update svg-sprite-loader dependencies, fix shipped with svg-baker@1.2.8
